### DISCUSSION TITLE
Weco - An Interchain Sensemaking Interface

### DISCUSSION THREAD

**Discussion by Miniminitrue** [akash-network/community#276] [AI - Artificial Intelligence]
Hello everyone,

My name is Gnoman, I'm a systems designer working on an experimental social network called https://www.weco.io who has been thinking through the role of the Cosmos in the future of governance. 

Weco is like reddit mixed with a mindmap. You have the traditional posting features, but you can connect any post to any other post. Then you have a holonic architecture which allows you to nest spaces (our equivalent of subreddits) within each other, and nest them in multiple locations simultaneously. We also have a conversation game called The Glass Bead Game which allows users to program conversations with media and time constraints, both synchronously in a WebRTC room and asychronously.

<img width="1492" alt="Screenshot 2023-07-21 at 16 09 48" src="https://github.com/akash-network/community/assets/34781840/42d23329-c968-4615-886a-a0f87e7f669b">

_In this screenshot, you can see a holonic tree of the Interchain. Spaces can be nested within each other, and in multiple places simultaneously._

<img width="1511" alt="Screenshot 2023-07-21 at 16 31 52" src="https://github.com/akash-network/community/assets/34781840/4427c366-9def-46fd-8c6a-256651eab582">

_In this screenshot, you can see the linkmap, which allows users to connect posts to other posts, and navigate through the interconnected graph._

My work in the Cosmos has involved thinking through the profound implications of holonics for governance. I wrote up my designs here, which won the 1st prize in the Better Future For The Cosmos competition.

https://docs.google.com/document/d/1lLlXFSMNUNnIU8K2ltVpT3aROcIosbkhuupQQFBubag/edit#

We're currently exploring using Weco as a sensemaking interface for the Interchain, using our advanced social thinking tools to help cosmonauts make sense of what is important to them. You can run polls, connect content together, have structured conversations with a timer, and use audio for greater expressivity. The holonics allow users to find the level of granularity of information within the Interchain they are looking for: all Cosmos SDK chains, one specific chain, a specific project on a specific chain etc.

With the launch of the Akash AI testnet, I've been inspired to think how we could integrate AI into our platform:

- What if when a post is being created, a user could prompt StableDiffusion or LLaMa2 to generate content for them?
- What if each space could have its own summary bots?
- What if different spaces could have their own fine-tuned models that would generate different results?
- What if Akash was hosting all the models, and users could pay in AKT or USDC for each compute?

We would love to collaborate with anyone from the community who is interested in this vision. We are a lean team of two (one system designer and one Web2 full stack dev) currently funded with a micro-grant by Jim Rutt (of https://www.jimruttshow.com/ and the leader of the Game B movement) with the goal of bringing us out of alpha and into beta, at which point he would migrate his Mighty Network of 10k users to Weco, as well as provide more funding. That means that we don't have a budget to hire someone now, but in the future will be hiring as we expand into the Interchain, including integrating with Akash in whatever ways we can.

I'm happy to organize a presentation call for whoever is interested where we could take you through the functionality of the platform and our vision.

P.S. For those who want to do a deeper dive:

Information about the holonic architecture can be found [here](https://github.com/wecollective/wiki/blob/main/new-wiki/holonics.md).

We did a [podcast about Weco](https://www.youtube.com/watch?v=gz-zstmKX70) here, where we discuss AI at one point, and I appeared on the Greenpill podcast [here](https://www.youtube.com/watch?v=gnYxwEjYLpM), which discusses the larger context of my work.

**Comment by bmenzalji** [comment]
Discussed with Gnoman in Discord.  This is not a fit at the time as their needs are to onboard an integrations engineer

"The needs are a developer who can collaborate with our dev to setup a model, eg stable diffusion or llama, connect it to our frontend, and provide a keplr connect for each user so they can make content by paying for a prompt."

This is not a fit for community fund initiatives at this time.

**Reply by Miniminitrue** [reply]
Don't feel this should be closed, as it prevents people who are interested in collaborating from seeing it. Our ask here on github is to the community, not to the community fund.
